src/styles/abstracts/_variables.scss

```
@@ -18,6 +18,8 @@ $tornado: #868686;
 $coral-red: #ff3c3c;
 $dove-gray: #646464;
 $azure-radiance: #0297fd;
+$mercury: #e1e1e1;
+$dark-charcoal: #333333;
```

This should also work fine.
```suggestion
$dark-charcoal: #333;
```